core/templates/dev/head/pages/exploration_editor/simple_editor_tab/questions/MultipleChoiceInputCheckerFilter.js

```diff
@@ -0,0 +1,42 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Filter that checks whether the interaction data for a
+ * MultipleChoiceInput interaction is consistent with the simple editor.
+ */
+
+oppia.filter('MultipleChoiceInputChecker', [function() {
```

Comment:
Because the process of adding a new interaction should, IMO, basically involve adding a new folder to questions/. The StatesToQuestions service is part of the "core" code and should not be touched.